packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -879,6 +880,9 @@ class _PopupMenuRoute<T> extends PopupRoute<T> {
   final BoxConstraints? constraints;
   final Clip clipBehavior;
   final AnimationStyle? popUpAnimationStyle;
+  final bool? _requestFocus;
+  @override
+  bool get requestFocus => _requestFocus ?? super.requestFocus;
```

Comment:
What do you think about instead adding a requestFocus parameter to Route and then just piping the requestFocus parameter here in _PopupMenuRoute through to `super`? I think that would be a more typical Flutter pattern and it would probably be easier to reason about why requestFocus has a certain value.